src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1751,6 +1790,56 @@ private void CheckForSolutionFile()
 				.FirstOrDefault()?.ItemPath;
 		}
 
+		public void CheckForBackgroundImage()
+		{
+			var iniPath = Path.Combine(WorkingDirectory, "desktop.ini");
+			if (!File.Exists(iniPath))
+				return;
+
+			// Read data
+			var lines = File.ReadLines(iniPath);
+			var keys = lines.Select(line => line.Split('='))
+								.Where(parts => parts.Length == 2)
+								.ToDictionary(parts => parts[0].Trim(), parts => parts[1].Trim());
+
+			// Image source
+			if (folderSettings.LayoutMode is not FolderLayoutModes.ColumnView && keys.TryGetValue("Files_BackgroundImage", out var backgroundImage))
```

Comment:
If I switch to the column layout with an image in the background, the image does not disappear.
Similarly, if I switch from the column layout to another layout, the image does not appear.

Reply:
I think this is because we only run this code when loading items

Reply:
We can add a `LayoutModeChangeRequested` event.

Reply:
Added